ibm/service/power/resource_ibm_pi_network_security_group_action.go

```diff
@@ -28,8 +28,8 @@ func ResourceIBMPINetworkSecurityGroupAction() *schema.Resource {
 		Importer:      &schema.ResourceImporter{},
 
 		Timeouts: &schema.ResourceTimeout{
-			Create: schema.DefaultTimeout(5 * time.Minute),
-			Update: schema.DefaultTimeout(5 * time.Minute),
+			Create: schema.DefaultTimeout(50 * time.Minute),
```

Comment:
Does it takes around 50 min to create/update?

Reply:
@hkantare There are edge cases requiring more time to enable a network security group in some workspaces